trino-connector/integration-test/trino-test-tools/trino-cascading-env/docker-compose.yaml

```diff
@@ -59,6 +88,7 @@ services:
       - ./init/trino-remote:/tmp/trino
       - ${GRAVITINO_TRINO_CONNECTOR_DIR:-/null}:/usr/lib/trino/plugin/gravitino
       - ${GRAVITINO_TRINO_CASCADING_CONNECTOR_DIR:-/null}:/usr/lib/trino/plugin/trino
+      - ${GRAVITINO_SERVER_DIR:-/null}:/opt/gravitino-server
```

Comment:
Why do we need to mount the gravtino server to the remote Trino

Reply:
```
cp /opt/gravitino-server/catalogs/jdbc-postgresql/libs/postgresql-42.7.0.jar  /usr/lib/trino/plugin/iceberg
cp /opt/gravitino-server/catalogs/jdbc-mysql/libs/mysql-connector-java-8.0.26.jar /usr/lib/trino/plugin/iceberg
```
because I want to copy the driver to iceberg plugin

Reply:
The driver also int the Trino original mysql and pg connector . You  can copy these drivers from that.  like `/usr/lib/trino/plugin/mysql`

Reply:
I check it in the Trino container, it seams there's drivers in /usr/lib/trino/plugin/mysql or /usr/lib/trino/plugin/postgresql.
```
[trino@54ba2a4a6ce8 /]$ ls /usr/lib/trino/plugin/mysql | grep mysql
mysql-connector-j-8.2.0.jar
trino-mysql-435-services.jar
trino-mysql-435.jar
[trino@54ba2a4a6ce8 /]$ ls /usr/lib/trino/plugin/postgresql | grep postgresql
postgresql-42.7.1.jar
trino-postgresql-435-services.jar
trino-postgresql-435.jar

```